packages/stark-build/config/build-utils.js

```diff
@@ -90,34 +101,39 @@ function getNbbAssetsConfig() {
 }
 
 /**
- * Returns assets set in .angular-cli.json file of the project.
+ * Returns assets set in angular.json file of the project.
  */
 function getApplicationAssetsConfig() {
 	const appConfig = getAngularCliAppConfig();
 
-	if (appConfig.assets) {
-		return getCopyWebpackPluginConfig(appConfig.assets);
+	if (
+		appConfig["architect"] &&
+		appConfig["architect"]["build"] &&
+		appConfig["architect"]["build"]["options"] &&
+		appConfig["architect"]["build"]["options"]["assets"]
+	) {
+		return getCopyWebpackPluginConfig(appConfig["architect"]["build"]["options"]["assets"]);
 	}
 
 	return [];
 }
 
 function getAngularCliAppConfig() {
-	const applicationAngularCliConfigPath = helpers.root(".angular-cli.json");
+	const applicationAngularCliConfigPath = helpers.root("angular.json");
 	if (fs.existsSync(applicationAngularCliConfigPath)) {
 		const angularCliConfig = require(applicationAngularCliConfigPath);
 		const cliConfig = ngCliUtils.validateAngularCLIConfig(angularCliConfig);
 		if (cliConfig) {
-			if (cliConfig["apps"] && cliConfig["apps"][0]) {
-				return cliConfig["apps"][0];
+			if (cliConfig["defaultProject"] && cliConfig["projects"][cliConfig["defaultProject"]]) {
```

Comment:
Again, I think we should just use the dot syntax whenever possible...

Reply:
done